src/styles/vars/_vars.scss

```diff
@@ -0,0 +1,13 @@
+$arrow-radius: 7px;
+$clock-width: 400px;
+$clock-height: 400px;
+$bdr-circle: 50%;
+$dot-width: 17px;
+$dot-height: 17px;
+
+$clr-red: red;
+$clr-gey: grey;
+$clr-blue: blue;
```

Comment:
It's better to give clearer names to variables. Not everyone can understand what clr or bdr mean